config/settings/development.yml

```diff
@@ -5,3 +5,4 @@ mcbg:
 system_authentication_token: "Ge32"
 log_level: debug
 gcp_api_key: please_change_me
+mapit_url: https://mapit.mysociety.org
```

Comment:
As the Mapit host is the same regardless of environment it can go in `config/settings.yml` and other environments will inherit this value.

Are we planning to have a Mapit secret per environment going forwards?

Reply:
No we're not actaully. Will make the changes.